presto-main/src/main/java/com/facebook/presto/server/SerializationUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.server;
+
+import com.facebook.airlift.json.ObjectMapperProvider;
+import com.facebook.presto.spi.function.BasicSignature;
+import com.facebook.presto.spi.function.SqlInvokedFunction;
+
+import java.io.IOException;
+
+public final class SerializationUtils
```

Comment:
Since we're doing any customized json serde here, we should just use the higher level API `JsonCodec` instead. This class won't be necessary.
```Java
public static JsonCodec<BasicSignature> JSON_CODEC = jsonCodec(BasicSignature.class);
```

Reply:
Removed the class